### PR TITLE
adds average rating, sorts airlines

### PR DESCRIPTION
**Pull request by MichaelSwartz**

### CODE REVIEW

File: app/models/airline.rb [hunk 1]
```diff
@@ -7,4 +7,17 @@ class Airline < ActiveRecord::Base
...
+    ratings_array = []
+    reviews.each do |review|
+      ratings_array << review.rating
+    end
+    ratings_array
+  end
+
+  def average_rating
+    sum = (ratings.inject (0) { |total, n| total + n }).to_f
+    sum / ratings.count
```
Comment (dreamingfurther) [inline app/models/airline.rb]: What happens if ratings.count is 0? I think you are leaving yourself open to a potential bug here if this method could ever get called when there are no ratings. I'd add unit test coverage for this edge case.

File: app/controllers/airlines_controller.rb [hunk 2]
```diff
@@ -2,7 +2,8 @@ class AirlinesController < ApplicationController
   before_action :authenticate_user!, except: [:index, :show]
 
   def index
-    @airlines = Airline.page(params[:page]).per(24)
+    airlines = Airline.all.sort { |x, y| y.average_rating <=> x.average_rating }
```
Comment (dreamingfurther) [inline app/controllers/airlines_controller.rb]: Could this be extracted into at least a class method? This is a lot of method chaining for the controller and its not super clear/easy to understand what the code is trying to do.

File: spec/features/airline_pagination_spec.rb [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+#
+#     FactoryGirl.create(:airline, name: "second_page_air")
+#
+#     visit airlines_path
+#
+#     click_on 'Next'
+#
+#     expect(page).to have_content("second_page_air")
+#   end
+# end
```
Comment (dreamingfurther) [inline spec/features/airline_pagination_spec.rb]: try not to commit large sections of commented code. Just delete it. It's in github. or if you want to keep it, keep it.
